src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/custom.py

```diff
@@ -314,9 +315,7 @@ def create_managed_disk(resource_group_name, disk_name, location=None,
 
     if size_gb is None and option == DiskCreateOption.empty:
         raise CLIError('usage error: --size-gb required to create an empty disk')
-
-    disk = Disk(location, disk_size_gb=size_gb, creation_data=creation_data,
-                account_type=sku, tags=(tags or {}))
+    disk = Disk(location, creation_data, (tags or {}), _get_sku_object(sku), disk_size_gb=size_gb)
```

Comment:
From experience in the network module, I'd highly recommend you use kwargs for all these params (like the original) and not use positional args. Inevitably some SDK change will be made that affects the order, this will mysteriously break, and you will wonder why because you are sure you tested it.  :smile: